nrfx/hal/nrf_nvmc.h

```diff
@@ -282,6 +321,24 @@ NRF_STATIC_INLINE void nrf_nvmc_nonsecure_mode_set(NRF_NVMC_Type *    p_reg,
 }
 #endif
 
+NRF_STATIC_INLINE void nrf_nvmc_word_write(uint32_t addr,
+                                           uint32_t value)
+{
+    *(volatile uint32_t *)addr = value;
```

Comment:
consider adding `NRFX_ASSERT` call to check for 4-byte address alignment. Applies to all functions